source/grid/tria_accessor.cc

```diff
@@ -1082,6 +1082,102 @@ set (const internal::Triangulation::TriaObject<structdim> &object) const
 
 
 
+template <>
+TriaAccessor<0,1,1>
+TriaAccessor<1,1,1>::vertex_accessor (const unsigned int i) const
+{
+  // Silence a warning
+  (void) i;
+  return TriaAccessor<0,1,1>();
+}
+
+
+
+template <>
+TriaAccessor<0,1,2>
+TriaAccessor<1,1,2>::vertex_accessor (const unsigned int i) const
+{
+  // Silence a warning
+  (void) i;
```

Comment:
Just don't name the argument in the argument list, and you don't need these two lines.